doc/source/cluster/cluster_under_construction/ray-clusters-on-vms/user-guides/launching-clusters/aws.md

```diff
@@ -0,0 +1,126 @@
+
+# Launching Ray Clusters on AWS
+
+This guide details the steps needed to start a Ray cluster on AWS.
+
+To start an AWS Ray cluster, you need to use the Ray Cluster Launcher with AWS Python SDK.
+
+
+## Install Ray Cluster Launcher
```

Comment:
Should we move this section to index.rst so that it's part of the common workflow before they jump into individual cloud providers?